sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/QueryResult.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using Azure.Core;
+using Azure.Core.Serialization;
+
+namespace Azure.DigitalTwins.Core
+{
+    internal partial class QueryResult<T>
```

Comment:
meaningful doc comments please. This is for your peers who read the code, not for web documentation.

What is this class for? What does it do? Why did we make it? :)